### PR TITLE
【代码迁移】任务41：On the Learning Property of Logistic 

### PR DESCRIPTION
**Pull request by xiandong20**
Snipaste_2024-10-07_19-19-07
On the Learning Property of Logistic and Softmax Losses for Deep Neural Networks

### CODE REVIEW

**Comment by nanzhaogang** [comment]
application/LGL-INR-master/LGL-INR-master/__pycache__/LGLINR.cpython-38.pyc
application/LGL-INR-master/LGL-INR-master/__pycache__/LGLINR_ms.cpython-38.pyc
这两个文件要删掉
